tests_zemu/tests/test.js

```diff
@@ -127,7 +129,17 @@ describe("Basic checks", function() {
         const expected_publicKey = "02beafa347af54948b214106b9972cc4a05a771a2573f32905c48e4dc697171e60";
 
         expect(resp.address).toEqual(expected_address_string);
+        console.log("Response address ", resp.address)
         expect(resp.publicKey.toString("hex")).toEqual(expected_publicKey);
+
+
+        // Now test if we can set the network version and get a testnet address
+        let res = await app.setAddressVersion(path, AddressVersion.TestnetSingleSig);
+
+        const response_t = await app.getAddressAndPubKey(path, true);
```

Comment:
`. getAddressAndPubKey` doesn't accept an argument, `true`?

Reply:
This is how It works right now:
```
app.setAddress(testnet)
app.getAddressAnPubkey(path)
// for setting mainnet back
app.setAddress(mainnet)
```
Further plans aim to use a common library for any app(most apps use the path for address versioning, stacks is an exception), but just talked to @jleni  and we can adapt the address-wise functions to include the AddressVersion param. It would become something like:
`app.getAddressAndPubkey(path, addressVersion)`
I guess It is preferable for you @kyranjamie, right?

Reply:
Hey @neithanmo, indeed it would be preferable to have both path and AddressVersion in the single call, as your example:

`app.getAddressAndPubKey(path: string, version: AddressVersion)`
`app.showAddressAndPubKey(path: string, version: AddressVersion)`

This would just be a little more of a predictable way of calling the methods as there's no chance, for example, other methods could mistakenly call the `setAddress` method to something else.